### ISSUE TITLE
aarch64

### ISSUE THREAD

**Issue by iotaka** [hpcugent/easybuild-framework#1419]
Hi,
I have an aarch64 for testing purposes. I installed easybuild via pip.

Then I tried to compile gcc 5.2, but it fails with the error

easybuild.tools.systemtools.SystemToolsException: Failed to determine CPU

/proc/cpuinfo return a ugly output

processor       : 7
Features        : fp asimd evtstrm
CPU implementer : 0x50
CPU architecture: 8
CPU variant     : 0x0
CPU part        : 0x000
CPU revision    : 0

**Comment by boegel** [comment]
@iotake: can you try running the `systemtools` part of the unit tests?

```
python -m test.framework.systemtools
```

**Comment by boegel** [comment]
test result in http://pastebin.com/aGu8qaVT

**Comment by boegel** [comment]
@iotaka: it seems like the CPU freq is simply not available, so try this as a trivial patch?
The tests made need some additional work to allow for a `None` value being returned by `get_cpu_speed`.

For the real fix, we may need something better though...
However, doesn't seem to be trivial: http://superuser.com/questions/406141/how-to-get-an-arm-cpu-clock-speed-in-linux.

```
diff --git a/easybuild/tools/systemtools.py b/easybuild/tools/systemtools.py
index b8c899b..8f3c589 100644
--- a/easybuild/tools/systemtools.py
+++ b/easybuild/tools/systemtools.py
@@ -229,7 +229,7 @@ def get_cpu_speed():
                 cpu_freq = float(res.group('cpu_freq'))
                 _log.debug("Found CPU frequency using regex '%s': %s" % (cpu_freq_regex.pattern, cpu_freq))
             else:
-                raise SystemToolsException("Failed to determine CPU frequency from %s" % PROC_CPUINFO_FP)
+                _log.debug("Failed to determine CPU frequency from %s", PROC_CPUINFO_FP)
         else:
             _log.debug("%s not found to determine max. CPU clock frequency without CPU scaling: %s" % PROC_CPUINFO_FP)
 
```

**Comment by boegel** [comment]
fix + updated test is tackled in #1421, let's follow up there
